python/taichi/lang/kernel_impl.py

```diff
@@ -476,6 +476,42 @@ def taichi_ast_generator():
         assert key not in self.compiled_functions
         self.compiled_functions[key] = self.get_function_body(taichi_kernel)
 
+    def get_torch_callbacks(self, v, is_ndarray, has_torch):
```

Comment:
My only nit: put `is_ndarray` in the end, so that we we call it, we can write `get_torch_callbacks(..., is_ndarray=True)`, so that ppl know what `True` `False` means... (Or we can use a more explicit variable name for T/F)

Reply:
SG, let me update it!

Reply:
Done :)